### ISSUE TITLE
Compatibility Issue with Food Funk

### ISSUE THREAD

**Issue by Roboowe** [GrowthcraftCE/Growthcraft-1.12#369]
While trying to make cheese I tried to insert a bucket of milk into a Cheese Vat but nothing happened. After some trouble shooting I found that items linked to [Food Funk](https://www.curseforge.com/minecraft/mc-mods/food-funk) cannot be used in Growthcraft machines. So far i've tested that items linked to Food Funk can not be used in the Cheese Vat or Pancheon and I assume this is the same for all other machines and Food Funk items. I'm not entirely sure why technical wise but I guess it's something to do with not recognizing the item.

Sorry in advance if this has already been mentioned.

**Comment by Roboowe** [comment]
Quick update:
After some further testing it is just putting liquids linked to Food Funk in the Cheese Vat or Pancheon that causes the problem.

**Comment by Alatyami** [comment]
That is correct, we do not have compatibility with Food Funk. We are currently working on the port to 1.15 and 1.16+ we can put this on the list, but will not be adding new features to 1.12 until that port is done.

**Comment by gatoborrachon** [comment]
> but will not be adding new features to 1.12 until that port is done.

(please kill my hope with a direct reply)
then you may fix the issue with [OpenJ9](https://github.com/GrowthcraftCE/Growthcraft-1.12/issues/363) in the future?
i ensure you that i can wait, but please don't let me with hope, that thing (hope) is a damn curse
